src/protocol/client.h

```diff
@@ -80,14 +95,15 @@ protocolClientMove(ProtocolClient *const this, MemContext *const parentNew)
 // Send noop to test connection or keep it alive
 void protocolClientNoOp(ProtocolClient *this);
 
-// Read a line
-String *protocolClientReadLine(ProtocolClient *this);
+// Get data put by the server
+PackRead *protocolClientDataGet(ProtocolClient *this);
+void protocolClientDataEndGet(ProtocolClient *this);
 
-// Read the command output
-const Variant *protocolClientReadOutput(ProtocolClient *this, bool outputRequired);
+// Put command to the server
+void protocolClientCommandPut(ProtocolClient *this, ProtocolCommand *command);
 
-// Write the protocol command
-void protocolClientWriteCommand(ProtocolClient *this, const ProtocolCommand *command);
+// Put data to the server
+void protocolClientDataPut(ProtocolClient *this, PackWrite *result);
```

Comment:
Why is the last parameter PackWrite *`result` here when in client.c it is `data`?

Reply:
Fixed in f011c0e3a46.